booklet.py

```diff
@@ -104,7 +110,9 @@ def generate_booklet(api, config, extra_events):
         end=end_date,
         emoji=config["tag_emoji"],
         published=published_string,
-        cover_dorms=[d for d in api["dorms"] if d != "All dorms"],
+        cover_dorms=sorted(
```

Comment:
Dorms should already be sorted in the API response, but sorting them again here probably can't hurt.

Reply:
I did this mainly because iHouse was being put at the end because of the lowercase i, but if there's a better way let me know!

Reply:
Should be fixed now (aka only sorting it once, I didn't see the `sorted()` in `process.py` earlier ahaha